CONTRIBUTING.rst

```diff
@@ -417,6 +417,18 @@ Then, run the following to install ``protoc``:
     sudo unzip -o $PROTOC_ZIP -d /usr/local 'include/*'
     rm -f $PROTOC_ZIP
 
+Alternatively, you can build protobuf files using Docker:
+
+.. code-block:: bash
+
+    pushd dev
+    DOCKER_BUILDKIT=1 docker build -t gen-protos -f Dockerfile.protos .
```

Comment:
Specified `DOCKER_BUILDKIT=1` to use `Dockerfile.protos.dockerignore` in this `docker build` (https://stackoverflow.com/a/57774684/6943581)